src/main/java/chatBot/bot/Storage.java

```diff
@@ -0,0 +1,62 @@
+package chatBot.bot;
+
+import chatBot.task.Task;
+
+import java.io.File;  // Import the File class
+import java.io.FileNotFoundException;  // Import this class to handle errors
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.FileWriter;
+
+public class Storage {
+    private File file;
+    private final String filePath;
+
+    Storage(String filePath) {
```

Comment:
Perhaps an access modifier could be added for the constructor here for the sake of completeness :)